### DISCUSSION TITLE
Unable to Create Account on strongswan.org

### DISCUSSION THREAD

**Discussion by thetuber** [strongswan/strongswan#764] [General]
I'm trying to create an account on strongswan.org so that I can edit the wiki. On https://wiki.strongswan.org/projects/strongswan is the "sign up" link https://wiki.strongswan.org/projects/strongswan/wiki/SignUp On that page is a "create an account" link https://wiki.strongswan.org/account/register which leads back to the original page.

**Comment by tobiasbrunner** [comment]
Yes, that site is basically in read-only mode (issue tracking has been moved here, the documentation will be moved to a different location, too, at some point in the future but we currently lack the resources to do so - existing users can make wiki edits, though). Please post suggestions for wiki edits here in the discussion forum or via email. Thanks.
